src/shared_modules/utils/loggerHelper.h

```diff
@@ -0,0 +1,200 @@
+/*
+ * Wazuh logging helper
+ * Copyright (C) 2015, Wazuh Inc.
+ * September 15, 2022.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#ifndef LOGGER_HELPER_H
+#define LOGGER_HELPER_H
+
+#include <functional>
+#include <iostream>
+#include <map>
+#include <memory>
+#include <mutex>
+#include <thread>
+#include <unordered_map>
+
+enum class LogLevel : int
+{
+    DEBUG_LOG = 0,
+    DEBUG_VERBOSE_LOG,
+    INFO_LOG,
+    WARNING_LOG,
+    ERROR_LOG
+};
```

Comment:
Just to force that constants values in C++ are the same as in C language, in this enum, I would force the value of each constant to the constant in C language enum

Reply:
Could you provide an example? Remember that `rsync` is a separate library and the shared code of the C code isn't included.

Reply:
Please, check the last changes.